pkg/docker/org.cockpit-project.cockpit-docker.metainfo.xml

```diff
@@ -2,7 +2,7 @@
 <component type="addon">
   <id>org.cockpit_project.cockpit_docker</id>
   <metadata_license>CC0-1.0</metadata_license>
-  <name>Docker Containers</name>
```

Comment:
According to [guidelines](https://www.patternfly.org/v4/design-guidelines/content/capitalization#capitalization-across-products) this should stay titlecase. There is at least one or two more examples  of the same problem. We might need just list of strings that should not be touched.